### DISCUSSION TITLE
Provide DNSCrypt, or DoT service as proxy interface, just like DoH.

### DISCUSSION THREAD

**Discussion by PeterDaveHello** [DNSCrypt/dnscrypt-proxy#2011] [Ideas]
We know that there's a local DoH support as https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Local-DoH mentioned, and I wonder if it's also possible to support DNSCrypt and DoT, especially DNSCrypt, so that this proxy can be used with more possibilities, for those clients support different encrypted DNS protocols, like Android 9+, they don't need to use plain-text based query to this proxy!

**Comment by bcookatpcsd** [comment]
This is a project that is also supported.. 

https://github.com/DNSCrypt/encrypted-dns-server

I don't think there will be support for DoT in these projects..
